content_schemas/README.md

```diff
@@ -0,0 +1,53 @@
+# GOV.UK content schemas
+
+This repo contains [JSON Schema](http://json-schema.org/) files and examples of the content that uses them on GOV.UK.
```

Comment:
```suggestion
This directory contains [JSON Schema](http://json-schema.org/) files and examples of the content that uses them on GOV.UK.
```

Reply:
Good spot, I've also removed / amended some other stuff in the README that's no longer relevant / needed updating